docs/content/preview/migrate/reference/datatype-mapping-oracle.md

```diff
@@ -45,20 +45,20 @@ The following table includes a list of supported data type mappings for migratin
 | BINARY_DOUBLE | DOUBLE PRECISION |
 | NUMBER | NUMERIC |
 | NUMBER(3) | SMALLINT |
-| NUMBER(3,2) | REAL |
+| NUMBER(3,2) | NUMERIC(3,2) |
 | NUMBER(2,7) | NUMERIC |
-| NUMBER(6,-2) | REAL |
-| BLOB | BYTEA | Currently, import data is not supported for BLOB/CLOB. Only the schema migration is allowed. |
-| CLOB | TEXT | Currently, import data is not supported for BLOB/CLOB. Only the schema migration is allowed. |
-| NCLOB | TEXT | Currently, import data is not supported for BLOB/CLOB. Only the schema migration is allowed. |
-| BFILE | BYTEA | Analysis of BFILES is still in progress and is currently unsupported. |
-| ROWID | OID | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| UROWID [(size)] | OID | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| SYS.AnyData | ANYDATA | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| SYS.AnyType | ANYTYPE | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| SYS.AnyDataSet | ANYDATASET | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| XMLType | XML | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| URIType | URITYPE | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
+| NUMBER(6,-2) | NUMERIC(6,-2) | Results in an error because negative scale is currently not supported. Refer to the [GitHub issue](https://github.com/yugabyte/yb-voyager/issues/779) for more details.
+| BLOB | BYTEA | Data is ignored during export. Currently, import data is not supported for BLOB/CLOB. Only the schema migration is allowed.  |
```

Comment:
Can we please replace from the second line, ie. `Currently...allowed`
with
```
Please use another mechanism to load the values.
```

Reply:
another mechanism - any tips or examples of what that might be?

Reply:
It's upto the user to decide that. They may use some other tool to do the task or manually import the values. We officially cannot suggest any such methods without proper research from our end.